GPIOOneWire/CPP/GpioOneWire.vcxproj

```diff
@@ -0,0 +1,191 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{5a9de0e6-1fa8-4f34-ab39-7473288becfa}</ProjectGuid>
+    <RootNamespace>GpioOneWire</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>

```

Comment:
In the readme, it says that it's using GpioChangeReader which is new to 19041, so shouldn't the min version be changed to 19041 to accommodate this?

Reply:
I have changed the MinimumPlatform to 15063 in readme, As we are using it consistently in all the samples.